Update link (#12721)
DataDog/integrations-core: docs/developer/architecture/win32_event_log.md

## Logs

> Events may alternatively be configured to be submitted as logs. The code for that resides

Reviewer: ```suggestion
Events may alternatively be configured to be submitted as logs. Use the code in the
```

> [here](https://github.com/DataDog/datadog-agent/tree/main/pkg/logs/internal/tailers/windowsevent).

Reviewer: ```suggestion
[here](https://github.com/DataDog/datadog-agent/tree/main/pkg/logs/internal/tailers/windowsevent).
```
```suggestion
[Datadog Agent GitHub repository](https://github.com/DataDog/datadog-agent/tree/main/pkg/logs/internal/tailers/windowsevent).
```
